README.md

```diff
@@ -23,6 +23,7 @@ This client supports the following Google Cloud Platform services at a [Beta](#v
 * [Google Cloud Dataproc](#google-cloud-dataproc-beta) (Beta)
 * [Google Cloud Natural Language](#google-cloud-natural-language-beta) (Beta)
 * [Google Cloud OsLogin](#google-cloud-oslogin-beta) (Beta)
+* [Google Cloud KMS](#google-cloud-kms-beta) (Beta)
```

Comment:
Small nit: Could we order this lexicographically?

Reply:
Done